pytest.ini

```diff
@@ -2,3 +2,7 @@
 addopts = -s
 log_cli=False
 log_level=DEBUG
+filterwarnings=
+  # FIXME: drop this once `pytest-cov` is updated.
```

Comment:
This  comment is excellent but insufficient:  it should include enough context so that a future developer can determine whether `pytext-cov` _has_ been updated (e.g., it should report either the current version number or the hoped-for target version number).